packages/realm/src/app-services/App.ts

```diff
@@ -50,8 +50,9 @@ export enum MetadataMode {
   Encryption = "encryption",
   /**
    * Do not persist {@link User} objects.
+   * @deprecated will be renamed to `InMemory`
    */
-  NoMetadata = "noMetadata",
+  NoMetadata = "inMemory",
```

Comment:
This will be an SDK breaking change if we don't allow users to keep using the old value until we bump our major. One option is to keep the old constant value (`noMetadata`) publicly, but send the new value to Core (`inMemory`).

Reply:
I will make it non-breaking.